web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -0,0 +1,371 @@
+            Study Owner (PI):
+            <Input
+              type="email"
+              id="owner"
+              {...register("owner")}
+              readOnly={true}
+              value={username}
+              inputClassName={styles.readonly}
+            />
+            <HelperText>
+              If you are not the study owner, contact the owner and ask them to fill out this form on their account.
+            </HelperText>
+          </Label>
+
+          <Label htmlFor="admin">
+            Study Administrator:
+            <Input type="email" id="admin" {...register("admin")} />
+            <HelperText>
+              {" "}
+              <strong>Must</strong> be a full UCL staff member, not honorary or affiliated
+            </HelperText>
+          </Label>
+
+          <Label htmlFor="controller">
+            Data Controller (organisation)*:
+            <Controller
+              name="controller"
+              control={control}
+              rules={{ required: "This field is required" }}
+              render={({ field }) => <Input {...field} type="text" id="controller" />}
+            />
+            {errors.controller && (
+              <Alert type="error">
+                <AlertMessage>{errors.controller.message}</AlertMessage>
+              </Alert>
+            )}
+          </Label>
+        </fieldset>
+
+        {/* second step */}
+        <fieldset className={getFieldsetClass(2)}>
+          <legend>Sponsorship & Approvals</legend>
+          <Label htmlFor="uclSponsorship" className={styles["checkbox-label"]}>
+            <input type="checkbox" id="uclSponsorship" {...register("uclSponsorship")} />{" "}
+            <span>
+              We will be seeking/have sought{" "}
+              <a href="https://www.ucl.ac.uk/joint-research-office/new-studies/sponsorship-and-grant-submissions">
+                UCL sponsorship
+              </a>{" "}
+              of this research
+            </span>
+          </Label>
+          <Label htmlFor="cag" className={styles["checkbox-label"]}>
+            <input type="checkbox" id="cag" {...register("cag")} />
+            <span>
+              {" "}
+              We will be seeking/have sought approval from the{" "}
+              <a href="https://www.hra.nhs.uk/about-us/committees-and-services/confidentiality-advisory-group/">
+                Confidentiality Advisory Group
+              </a>{" "}
+              for this research
+            </span>
+          </Label>
+          {showCagRef && (
+            <Label htmlFor="cagRef">
+              Confidentiality Advisory Group Reference
+              <Input type="text" id="cagRef" {...register("cagRef")} />
+            </Label>
+          )}
+
+          <Label htmlFor="ethics" className={styles["checkbox-label"]}>
+            <input type="checkbox" id="ethics" {...register("ethics")} />
+            We will be seeking/have sought Research Ethics Committee approval for this research
```

Comment:
@finleybacon I assume this is a requirement for some studies? can/should we enforce this?